tox.ini

```diff
@@ -0,0 +1,21 @@
+[tox]
+skipsdist = true
+envlist = tests, flake8
```

Comment:
Let's split tests into desktop and mobile, so we'll have three environments:

```ini
envlist = desktop, mobile, flake8
```

Then we can create a `[testenv:mobile]` section for overrides specific to the mobile suite. Because of the differences it may be that we can't run both desktop and mobile using a single command `tox` for now.